images/opentelemetry/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~  Copyright 2022 Red Hat, Inc., and individual contributors
```

Comment:
Nitpicking, maybe but shouldn't it be 2023? Again, nothing blocking anything, but just for my information, are the headers generated or do we need to copy/paste them manually?

Reply:
People generally have these set up in their IDE. IntelliJ supports something like $day.year which gets replaced with the current year though :-)

Reply:
That was probably a copy-and-paste class. My IDE collapses the header comments, so I missed that. I'll get that updated and verify any others.